vimrc

```diff
@@ -68,6 +133,8 @@ endif
 "  endfor
 "endfor
 
+autocmd FocusLost * silent! wall
```

Comment:
Based on some googling, it looks like this automatically saves when vim loses focus. I could easily see that causing data loss if someone didn't know it was there and assumed default vim behavior.

Reply:
That is indeed the intent of that setting. I _think_ it really only works in a GUI though. I could see an argument to put it into the existing `if has("gui_running")` check or having that just need to be pulled out into my personal `.vimrc.local`. Thoughts?

Reply:
I think .vimrc.local would be the way to go for this one.

Reply:
done